guides/common/modules/ref_capsule-storage-requirements.adoc

```diff
@@ -5,45 +5,35 @@
 The following table details storage requirements for specific directories.
 These values are based on expected use case scenarios and can vary according to individual environments.
 
+ifdef::katello,satellite[]
 The runtime size was measured with {RHEL} 6, 7, and 8 repositories synchronized.
```

Comment:
```suggestion
The runtime size was measured with {EL} 6, 7, and 8 repositories synchronized.
```

Reply:
I explicitly didn't change this. It refers to a benchmark which synced real RHEL and that resulted in a certain size. Other EL versions may result in different sizes. We have seen that the number of changelogs that was in the RPM metadata made real differences in size, so this is the better guidance IMHO.